mlibrary/catalog-browse
# [LIBSEARCH-855] Create a new Catalog and Online Journal search field for "LC Subject Starts With"

# Overview
> * **Label (in field drop-down):**  LC Subject starts with
> * **Search Hint (below search box):** Enter words or phrases to see subjects that start with them. (e.g. Baldwin, James; sociology dictionaries)
> * Placed immediately after Subject in simple and advanced search menus

This pull request closes [LIBSEARCH-855](https://mlit.atlassian.net/browse/LIBSEARCH-855).

## Anything else?
The dropdown options did not match what is currently in Search. The options have been updated so both sites will match.

## Testing
- Make sure the PR is consistent in these browsers:
- Check if the new option appears.
- Make sure the dropdown options match Search.
